src/helpers/S3Helper.php

```diff
@@ -78,18 +78,18 @@ public static function uploadFileFromModel(Model $model, string $filePath, strin
 
 	/**
 	 * Метод для удаления файлов
-	 * @param int $storageId
+	 * @param CloudStorage|int $storage
 	 * @param string|null $bucket
 	 * @return int|null
 	 * @throws Throwable
 	 */
-	public static function deleteFile(int $storageId, ?string $bucket = null):?int {
-		/** @var CloudStorage $storage */
-		if (null === $storage = CloudStorage::find()->where(['id' => $storageId])->active()->one()) return null;
-		$storage->deleted = true;
-		$storage->save();
+	public static function deleteFile(CloudStorage|int $storage, ?string $bucket = null):?int {
+		$storageModel = $storage;
+		if (is_int($storage) && null === $storageModel = CloudStorage::find()->where(['id' => $storage])->active()->one()) return null;
```

Comment:
```suggestion
		if (null === $storageModel = (is_int($storage))
				?CloudStorage::find()->where(['id' => $storage])->active()->one()
				:$storage) return null;
```

Как-то с двумя типами не очень красиво получается. Постарался переписать код, чтобы без лишних присваиваний.